Week-1/Assignment_1.2/index.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="App.css" />
+    <link rel="stylesheet" href="https://maxst.icons8.com/vue-static/landings/line-awesome/line-awesome/1.3.0/css/line-awesome.min.css">
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+    <link
+</head>
+<body>
+    <div class='App flx-cl'>
+        <!-- {/* MAIN-SECTION */} -->
+        <header class='flx-rw main-sec'>
+  
+          <!-- {/* NAVBAR */} -->
+          <nav class='navbar bck-clr-gry'>
+            <div class='nav-icn-cont flx-cl flx-1'>
+              <div class='nav-icns flx-1'>
```

Comment:
follow BEM naimg convention throughout https://getbem.com/naming/

Reply:
Sure @anilrao316 will look after this and follow once I get hold of the principals.